src/SignalR/clients/csharp/Client.SourceGenerator/src/HubProxyAttributes/HubProxyAttributes.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    /// <summary>
+    /// Place this attribute on a method with the following syntax:
```

Comment:
Should we steal from the logging description:

https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerMessageAttribute.cs#L9

> Provides information for generating a strongly typed SignalR client. Place this attribute on a method with the following syntax:

Reply:
Wait are these attributes in the source generator? How does that work since you don't reference the source generator from code?

Reply:
>Wait are these attributes in the source generator?

Yes.

>How does that work since you don't reference the source generator from code?

You reference the source generator package, and normally the package would contain just an analyzer folder, but this one contains both an analyzer folder and a netstandard2.0 folder with a consumable dll.